3_visualize/src/report-wqp-salinity-data.Rmd

```diff
@@ -10,26 +10,23 @@ output: html_document
 
```

Comment:
I'd definitely be open to that - I initially created the markdown doc as a way to show some draft figures and document some notes. My 2 cents is that most of the code within the markdown doc would represent such a small target that it's not worth porting over to a separate function (e.g. code that tallies censored/preliminary data within the harmonized WQP dataset). 

I think it's worth moving some code into functions for figures, though. I'll suggest that I start by moving the leaflet map code into a separate function. Your comment here made me rethink the location of the code that cleans up the different crs within the site list. I've since moved that code from the markdown doc to `2_process/create_site_list.R` instead, which will hopefully make downstream tasks like snapping site locations to PRMS flowlines easier.